app/routes/api.js

```diff
@@ -39,6 +39,47 @@ router.get('/fact', (req, res) => {
     }
 });
 
+// API endpoint to add a new fact to the database.
+router.post('/fact', (req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+    const { userId, content, discovery_date, note } = req.body;
+
+    // v no longer fails foreign key constraint
+    const submitter_id = userId || 'zzz3737';
```

Comment:
Could you define a constant for the anonymous user ID? So we can reuse and change it later.

Reply:
should I just define it in the same file?